### PR TITLE
Added new allocation status "Active (Needs Renewal)"

### LINKED ISSUE #120: Update ColdFront text "Expired" to "Active (Needs Renewal)" #588
Copied from nerc-project/operations#588, we want to rename the allocation status "Expired" to "Active (Needs Renewal)"

### PR DESCRIPTION
**Pull request by QuanMPhm**
Closes #120. When any Coldfront expires, the allocation's status will change to "Active (Needs Renewal)" instead of "Expired"

### CODE REVIEW

**Comment by jtriley** [comment]
@QuanMPhm we also need to include the patch in the Dockerfile:

https://github.com/nerc-project/coldfront-nerc/blob/main/Dockerfile#L30

**Review by jtriley — CHANGES_REQUESTED** [review]
@QuanMPhm we also need to include the patch in the Dockerfile:

https://github.com/nerc-project/coldfront-nerc/blob/main/Dockerfile#L30

File: patches/03_add_active_needs_renewal_status.patch [hunk 1]
```diff
@@ -0,0 +1,26 @@
+diff --git a/coldfront/core/allocation/management/commands/add_allocation_defaults.py b/coldfront/core/allocation/management/commands/add_allocation_defaults.py
+index 556eeb2..ca5c4fc 100644
+--- a/coldfront/core/allocation/management/commands/add_allocation_defaults.py
++++ b/coldfront/core/allocation/management/commands/add_allocation_defaults.py
+@@ -16,7 +16,7 @@ class Command(BaseCommand):
+             'Attribute Expanded Text'):
+             AttributeType.objects.get_or_create(name=attribute_type)
+ 
+-        for choice in ('Active (Needs Renewal)', 'Active', 'Denied', 'Expired',
```
Comment (jtriley) [inline patches/03_add_active_needs_renewal_status.patch:9]: This patch looks to be the reverse of the change you're attempting to make...

**Review by jtriley — APPROVED** [review]

**Comment by jtriley** [comment]
@QuanMPhm LGTM and approved but would like to wait for @knikolla review before merging. Thanks!!

**Review by knikolla — APPROVED** [review]
Looks good to me.

@QuanMPhm Have you had a chance to test that the `add_allocation_defaults` function works correctly when executed on a ColdFront that already has the allocation defaults?

**Comment by QuanMPhm** [comment]
@knikolla When the patched command is ran on a Coldfront with the old defaults, the new `Active (Needs Renewal)` allocation status is added. I did not see any errors.

**Comment by joachimweyl** [comment]
@QuanMPhm what are next steps to get this merged?

**Comment by QuanMPhm** [comment]
@joachimweyl I suppose I'll merge it now. I was just waiting for someone with more seniority to merge it.

**Pull request merged**